dashboard/app/views/plc/enrollment_evaluations/preview_assignments.html.haml

```diff
@@ -0,0 +1,15 @@
+- course_unit = @enrollment_unit_assignment.plc_course_unit
+
+You will be assigned these modules
+- @enrolled_modules.each do |enrolled_module|
+  %li
+    = enrolled_module.name
+
+%br
+You can change your course enrollment here
+= form_tag(action: 'confirm_assignments') do
+  - options = course_unit.plc_learning_modules.pluck(:name, :required, :id).map {|name, required, id| ["#{name}#{' - Required' if required}", id]}
+  = select_tag 'learning_module_ids', options_for_select(options, selected: @enrolled_modules.pluck(:id), disabled: course_unit.plc_learning_modules.where(required: true).pluck(:id)), {multiple: true, size: [options.size, 25].min, style: 'width: 100%'}
```

Comment:
This is the second usage of the `size: [options.size, 25].min` logic which makes this a good candidate for extracting a `max_options_select_tag` helper.  See [`boolean_check_box`](https://github.com/code-dot-org/code-dot-org/blob/d29dcf90cf7654371dcede0f39c1d622d9f5bfa6/dashboard/app/helpers/levels_helper.rb#L491-L493) for a good example.